trade-finance-manager-api/src/v1/helpers/amendment.helpers.js

```diff
@@ -543,6 +543,7 @@ const calculateAcbsUkefExposure = (payload) => {
  */
 const formatCoverEndDate = (payload) => {
   if (payload?.coverEndDate) {
+    // Should this be covered by 6998?
```

Comment:
I think this comment wants the dates to be stored in the same way - not sure we have a story/task for it on Jira?

Reply:
I don't think it should be covered by this ticket, but it needs a ticket assigning in the TODO